packages/fxa-admin-server/src/config.ts

```diff
@@ -79,6 +79,18 @@ const conf = convict({
     doc: 'Sentry DSN for error and log reporting',
     env: 'SENTRY_DSN',
     format: 'String'
+  },
+  hstsEnabled: {
+    default: true,
+    doc: 'Send a Strict-Transport-Security header',
+    env: 'HSTS_ENABLED',
+    format: Boolean
+  },
+  hstsMaxAge: {
+    default: 31536000, // a year
+    doc: 'Max age of the STS directive in seconds',
+    // Note: This format is a number because the value needs to be in seconds
+    format: Number
```

Comment:
I think we need this in `fxa-admin-panel` as well. I have the config set up for it but didn't add the middleware.

Reply:
Or... maybe not? I see the issue only lists the admin-server.